src/main/webapp/WEB-INF/applicationContext.xml

```diff
@@ -2,10 +2,13 @@
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xmlns:context="http://www.springframework.org/schema/context"
        xsi:schemaLocation="http://www.springframework.org/schema/beans
-  http://www.springframework.org/schema/beans/spring-beans.xsd
-  http://www.springframework.org/schema/context
-  http://www.springframework.org/schema/context/spring-context.xsd"
->
-    <bean id="greetingService" class="org.noteskeeper.server.GreetingService"/>
+           http://www.springframework.org/schema/beans/spring-beans-2.5.xsd
```

Comment:
XML-based Spring configs are legacy. Let's use Java configs - https://docs.spring.io/spring/docs/current/spring-framework-reference/htmlsingle/#beans-java. As for mkyong and other 3rd party sites, Spring has pretty awesome own docs and a lot of official examples on github. They  are trying to keep them always up to date. So it is much better to use.